docs/2.5. APIs - Query Parameters.md

```diff
@@ -218,18 +237,20 @@ Payload Resources
 ```
 
 ***Notes***
-* Whilst both 'since' and 'until' are specified, as this server example has a default paging limit of '10', the 'since' parameter must take precedence. As a result of this the value of X-Paging-Until is lower than requested and a further request must be made to retrieve any remaining data.
+
+- Whilst both 'since' and 'until' are specified, as this server example has a default paging limit of '10', the 'since' parameter  takes precedence. As a result of this the value of X-Paging-Until is lower than requested and a further request MUST be made to retrieve any remaining data.
 
 ### Edge Cases
 
-When a client requests data which falls at the extreme ends of the stored data set it may be less clear what values should be returned in the 'X-Paging-Limit' and 'X-Paging-Since' headers. The following examples are intended to clarify the expectation in these cases.
+When a client requests data which falls at the extreme ends of the stored data set it can be less clear what values SHOULD be returned in the 'X-Paging-Limit' and 'X-Paging-Since' headers. The following examples are intended to clarify these cases.
```

Comment:
This doesn't read like a requirement, how about:
```suggestion
When a client requests data which falls at the extreme ends of the stored data set it can be less clear what values ought to be returned in the 'X-Paging-Limit' and 'X-Paging-Since' headers. The following examples are intended to clarify these cases.
```

Reply:
...ought to be..." a bit of a strange construction for non-native speakers. What about "...are expected to be...

Reply:
Agreed